cl/corpus_importer/management/commands/recap_into_opinions.py

```diff
@@ -0,0 +1,124 @@
+from asgiref.sync import async_to_sync
+from django.core.management import BaseCommand
+from django.core.paginator import Paginator
+from django.db import transaction
+
+from cl.lib.command_utils import logger
+from cl.lib.microservice_utils import microservice
+from cl.search.models import (
+    SOURCES,
+    Court,
+    Docket,
+    Opinion,
+    OpinionCluster,
+    RECAPDocument,
+)
+
+
+def import_opinions_from_recap(court=None, total_count=0):
+    """Import recap documents into opinion db
+
+    :param court: Court ID if any
+    :param total_count: The number of new opinions to add
+    :return: None
+    """
+    count = 0
+    if not court:
+        courts = Court.objects.filter(jurisdiction=Court.FEDERAL_DISTRICT)
+    else:
+        courts = Court.objects.filter(id=court)
```

Comment:
Using pk is better b/c `id` is a reserved word in Python:

```suggestion
        courts = Court.objects.filter(pk=court)
```